src/atoms/typography/heading.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { HeadingPriority } from "../constants/heading-priority";
+
+// -----------------------------------------------------------------------------------------
+// #region Interfaces
+// -----------------------------------------------------------------------------------------
+
+export interface HeadingProps {
+    children?: any;
```

Comment:
Nitpick - prefer wrapping this interface with `PropsWithChildren` instead of manually defining `children`